test/libsolidity/syntaxTests/storageLayoutSpecifier/hex_address.sol

```diff
@@ -1,3 +1,3 @@
 contract C layout at 0xdCad3a6d3569DF655070DEd06cb7A1b2Ccd1D3AF {}
 // ----
-// TypeError 6396: (21-63): The base slot of the storage layout must evaluate to a rational number.
+// TypeError 1763: (21-63): The base slot of the storage layout must evaluate to an integer. The type of the supplied expression is address.
```

Comment:
I like this one. Without the extra detail it might be hard for the user to realize that we interpret the literal as an address here.